mks/model/stuff.py

```diff
@@ -289,23 +313,60 @@ def get_kinderen(self):
             {'name': 'geslachtsnaam', 'parser': self.to_string},
             {'name': 'geslachtsaanduiding', 'parser': self.to_string},
             {'name': 'geboortedatum', 'parser': self.to_date},
+            {'name': 'inp.geboorteplaats', 'parser': self.to_date, 'save_as': 'geboorteplaats'},
```

Comment:
Also here `self.to_date` seems incorrect for a non date value

Reply:
copy pasta mistake :speak_no_evil: